File: Modul 4/Soal 4/PRAK304-2210817220018-Piola Evania.py
for i in range(5):  
    a = int(input())

    if(a == 0):
        print("Nol\n")
    elif(a >= 1 and a <= 9):
        print("Satuan\n")
    elif(a >= 11 and a <= 19):
        print("Belasan\n")
    elif(a == 10 or a >= 20 and a <= 99):
        print("Puluhan\n")
    else:
        print("Anda Menginput Melebihi Limit Bilangan\n")

File: Modul 4/Soal 4/PRAK304-2210817220018-Piola Evania.c
#include <stdio.h>

int main()
{
    for(int i=1; i<6; i++){
    int a;
    scanf("%d", &a);

        if(a==0){
            printf("Nol\n\n");
        }
        else if(a>0 && a<=9){
            printf("Satuan\n\n");
        }
        else if(a>=11 && a<=19){
            printf("Belasan\n\n");
        }
        else if(a==10 || a>=20 && a<=99){
            printf("Puluhan\n\n");
        }
        else{
            printf("Anda Menginput Melebihi Limit Bilangan\n\n");
        }
    }
}
